integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -180,10 +191,19 @@ async def _get_item_content(
                     method="GET", url=items_url, params=items_params
                 )
             ).content
-
+        except HTTPStatusError as e:
+            general_err_msg = f"Couldn't fetch file {file_path} from repo id {repository_id}: {str(e)}. Returning empty file."
+            if e.response.status_code == 404:
+                logger.warning(
+                    general_err_msg
+                    + f" This may be because the repo {repository_id} is disabled."
```

Comment:
use fstring for the whole log, you are combining two concatenation methods instead of continuing the convention we have.